pkg/asset/machines/master.go

```diff
@@ -264,6 +264,7 @@ func (m *Master) Machines() ([]machineapi.Machine, error) {
 	gcpapi.AddToScheme(scheme)
 	libvirtapi.AddToScheme(scheme)
 	openstackapi.AddToScheme(scheme)
+	gcpprovider.SchemeBuilder.AddToScheme(scheme)
```

Comment:
this is already being set [here](https://github.com/openshift/installer/blob/1a78ab5e12aa9295b20c789c0e42c0760bb753c1/pkg/asset/machines/master.go#L264) [api](https://www.godoc.org/github.com/openshift/cluster-api-provider-gcp/pkg/apis#AddToScheme)

Reply:
ack thx, I think I added this initially when I got the error about unrecognized type, but it's likely that only the group was wrong. Will remove.